mmaction/models/heads/lfb_head.py

```diff
@@ -0,0 +1,122 @@
+import warnings
```

Comment:
Maybe you can modify and directly use the AVARoIHead? Seems the only difference is that you also pass kwargs to the shared head. Maybe no need to add a `lfb_head`

Reply:
yeah，then I don't need to import mmdet